entry/src/main/resources/base/layout/ability_main.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DirectionalLayout
+    xmlns:ohos="http://schemas.huawei.com/res/ohos"
+    ohos:height="match_parent"
+    ohos:width="match_parent"
+    ohos:background_element="$media:back"
+    ohos:orientation="vertical">
+
+    <Text
+        ohos:id="$+id:text_helloworld"
```

Comment:
please change the text's id to a meaningful id name. Also the value of the text  "helloworld" can be changed to meaningful string value.

Reply:
Done